tests/unit/test_BufferedSingerStream.py

```diff
@@ -75,6 +76,72 @@ def test_add_record_message__invalid_record():
     assert [] == missing_sdc_properties(singer_stream)
 
 
+SIMPLE_MULTIPLE_OF_VALID_SCHEMA = {
+    'properties': {
+        'multipleOfKey': {
+            'type': 'number',
+            'multipleOf': Decimal('1e-15')
+        }
+    }
+}
+
+SIMPLE_MULTIPLE_OF_INVALID_SCHEMA = {
+    'properties': {
+        'multipleOfKey': {
+            'type': 'number',
+            'multipleOf': 1e-15
+        }
+    }
+}
+
+def test_add_record_message__multipleOf():
+    stream_name = 'test'
+    singer_stream = BufferedSingerStream(stream_name,
+                                         deepcopy(SIMPLE_MULTIPLE_OF_VALID_SCHEMA),
+                                         [])
+
+    multiple_of_values = ['1', '2', '3', '4', '5', '1.1', '2.3', '1.23456789', '20', '100.1']
+
+    for value in multiple_of_values:
+        singer_stream.add_record_message(
+            {
+                'type': 'RECORD',
+                'stream': stream_name,
+                'record': {'multipleOfKey': Decimal(value)},
+                'sequence': 0,
+                RAW_LINE_SIZE: 100
+            }
+        )
+
+    assert not singer_stream.peek_invalid_records()
+    assert singer_stream.count == len(multiple_of_values)
+    assert [] == missing_sdc_properties(singer_stream)
```

Comment:
@davicorreiajr it's this line, we can remove it same as the other, sorry for the confusion and appreciate your patience!

Reply:
no worries, fixing now